bash/02_build_attribute.sh

```diff
@@ -152,7 +152,7 @@ echo
 
 # geometry cleaning -- lines to polygons and all geoms to multi
 echo 'Cleaning geometries: lines to polygons and geoms to multi'
-psql $BUILD_ENGINE -f sql/attributes_geomclean.sql
+psql $BUILD_ENGINE  -q  -f sql/attributes_geomclean.sql
```

Comment:
Remove quiet flag.  We want to see what's happening here.

Reply:
This issue with the output from this file is that it would print to stdout each geometry it fixed, filling the logs up with hundreds of near-duplicate lines. Before the quiet flags were added and pandas updates made the build attributes step log had 11.7k lines. I think this is one reason we were missing error messages

Reply:
Oh okay.  Then leave as is.